_scripts/instruction-widget/install.js

```diff
@@ -131,18 +137,24 @@ module.exports = function(context) {
     context.cbauto = false;
     if (context.webserver == "apache") {
       context.package = "certbot-apache";
+    } else if (context.webserver == "nginx") {
+      context.certonly = true;
```

Comment:
It does, but trying to actually install it didn't work for me. I emailed the maintainer, but he hasn't responded about that yet. Can you get it to install?